physical_property/logP/analysis/SAMPL7-logP-method-map.csv

```diff
@@ -0,0 +1,37 @@
+receipt_id,name,category,reassigned_category,physical_method_details
+REF00,ChemAxon,Empirical,Empirical,
+NULL0,mean clogP of FDA approved oral drugs (1998-2017),Empirical,Empirical,
+HC032,Chemprop,Empirical,Empirical,
+0680E,TFE-prediction-Attentive FP,Empirical,Empirical,
+0A608,GROVER,Empirical,Empirical,
+063A2,COSMO-RS,Physical,Physical (QM),QM
+00EDA,ffsampled_deeplearning_cl1,Empirical,Empirical,
+096BB,ffsampled_deeplearning_cl2,Empirical,Empirical,
+08F3F,MD-EE-MCC (GAFF-TIP4P-Ew) ,Physical,Physical (MM),MM (GAFF/TIP4P-EW)
+004AF,Ensemble prediction of TFE,Empirical,Empirical,
+2DF02,Ensemble prediction of TFE,Empirical,Empirical,
+08DA5,TFE-prediction-solvation-b3lypd3,Physical,Physical (QM),QM
+02F1F,ClassicalGSG,Empirical,Empirical,
+4K631,ClassicalGSG,Empirical,Empirical,
+43M66,ClassicalGSG,Empirical,Empirical,
+847L9,ClassicalGSG,Empirical,Empirical,
+0357F,TFE-prediction-method-IEFPCM/MST,Physical,Physical (QM),QM
+047D4,EC_RISM_wet,Physical,Physical (QM),QM
+D4406,TFE-MD-neat-octanol,Physical,Physical (MM),MM (GAFF/TIP4P)
+065C7,TFE-MD-wet-octanol,Physical,Physical (MM),MM (GAFF/TIP4P)
+07B59,NES-1,Physical,Physical (MM),MM (GAFF2/OPC3)
+F46P0,NES-1,Physical,Physical (MM),MM (GAFF2/OPC3)
+C4A03,NES-1,Physical,Physical (MM),MM (GAFF2/OPC3)
+09F0E,RayLogP三_QSPR_Mordred2D_TPOT-AutoML,Empirical,Empirical,

```

Comment:
(Alternatively maybe just use the submission NUMBER instead of the method map stuff. Main point: I can't remember to refer to submission "02BF8" for more than two seconds, but I can refer to "TFE-SM12-vacuum-opt" (and the latter tells me something about what the method was) and I can also remember to talk about submission 27. In other words, there is no good reason to introduce extra sets of hexadecimal characters to refer to the submissions, when we could just as well use their names or submission IDs.)

Reply:
The method map has been removed. Everything is based off of method names now.